core-plugins/docs/Deduplicate-batchaggregator.md

```diff
@@ -12,6 +12,11 @@ The `first` and `last` are deprecated and will be removed in later versions of t
 The `any` logical function will match any of the records where the value for the specified field is non-null.
 If not found, a record where the value of this field is null will be returned.
 
+Deduplicate stages are now eligible to execute in BigQuery when BigQuery ELT Transformation Pushdown is enabled in 
+your pipeline. Said Stages will be executed in BigQuery when a preceding stage has already been executed in BigQuery 
```

Comment:
@sanjanasandeep instead of "Said Stages" can we say "Deduplicate stages...."

Reply:
Can you add a header for BigQuery ELT Transformation Pushdown? Just so it's a separate section of the doc.